src/backend/web/routes/query.js

```diff
@@ -8,8 +8,8 @@ const router = express.Router();
 
 router.get('/', validateQuery(), async (req, res) => {
   try {
-    const { text, filter } = req.query;
-    res.send(await search(text, filter));
+    const { text, filter, page, perPage } = req.query;
```

Comment:
These need to get added to our `validateQuery` validation logic

Reply:
added, let me know if I did it right.